### ISSUE TITLE
Ability to change guidlineBase Width & Height

### ISSUE THREAD

**Issue by zek** [nirsky/react-native-size-matters#28]
In my project, designer uses Iphone 8 scale which is 375x667. I guess we could be able to change width&height rate or I should tell to the designer use 350x680? What do you think about it?

~~~
const guidelineBaseWidth = 350;
const guidelineBaseHeight = 680;
~~~

![image](https://user-images.githubusercontent.com/3463291/56446075-36fe4600-6309-11e9-85ce-f1bc10ff33a9.png)

![image](https://user-images.githubusercontent.com/3463291/56446068-31a0fb80-6309-11e9-9dd9-65f8029d9691.png)

**Comment by Balasnest** [comment]
This should be generic.

**Comment by nirsky** [comment]
Hey @zek, the difference between 375x667 to 350x680 is not very significant.
If it was me - I'd use the iPhone 8 design the designer gave me as is.

If it really bothers you, you have 2 other options:
* Like you mentioned - ask the designer to design for 350x680 screen.
* Fork this lib and publish to npm with the designated dimensions.

@Balasnest - unfortunately it's impossible to make it generic since StyleSheets are usually static, meaning it's impossible to change it on runtime.
I examined many different solutions for this but they all require some peer dependencies like `react-native-config` or `react-native-dotenv`, which I don't intent to enforce.
If you have any suggestion on how to make it generic I'd like to know.

**Comment by nirsky** [comment]
Ok, so after many issues regarding this and many attempts to come up with the best solution, I came up with something nice.
Check PR https://github.com/nirsky/react-native-size-matters/pull/29.

It's already merged and published under version `0.2.0`.
Enjoy!
